File: BirdwatcherIoTClient/BirdLog.py
class BirdLog:
    
    def __init__(self, Timestamp, Temperature, Picture, Location_Latitude, Location_Longitude, UserGUID, Birds):
        self.Temperature = Temperature
        self.Picture = Picture
        self.Location_Latitude = Location_Latitude
        self.Location_Longitude = Location_Longitude
        self.UserGUID = UserGUID
        self.Birds = Birds

File: BirdWatcherMobileApp/BirdWatcherMobileApp/Models/BirdLog.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace BirdWatcherMobileApp.Models
{
    public class BirdLog
    {
        public int birdLogID { get; set; }
        public DateTime timestamp { get; set; }
        public double temperature { get; set; }
        public double location_latitude { get; set; }
        public double location_longitude { get; set; }
        public string userGUID { get; set; }
        public string picture { get; set; }
        public List<int> birds { get; set; }
    }
}
